src/main.js

```diff
@@ -265,7 +265,7 @@ export default {
 
     if (Array.isArray(data)) {
       data.forEach((d) => {
-        wrapper.appendChild(this.createNodeByType(content));
+        wrapper.appendChild(this.createNodeByType(data));
```

Comment:
Shouldn't this be `d` instead of `data` ?

Reply:
Let me also write a sample data in the dev page